pkg/k8sutil/create.go

```diff
@@ -165,3 +172,59 @@ func parseJSONManifest(data []byte) ([]manifest, error) {
 	}
 	return manifests, nil
 }
+
+// CreateOrUpdateNamespace creates the release namespace or updates the namespace
+// if it already exists.
+func CreateOrUpdateNamespace(ns Namespace, nsclient corev1typed.NamespaceInterface) error {
+	if ns.Name == "" {
+		return fmt.Errorf("namespace name can't be empty")
+	}
+
+	_, err := nsclient.Get(context.TODO(), ns.Name, metav1.GetOptions{})
+	if err != nil {
```

Comment:
I think we can shorten here, the line shouldn't be too long.
```suggestion
	if _, err := nsclient.Get(context.TODO(), ns.Name, metav1.GetOptions{}); err != nil {
```